content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md

```diff
@@ -31,7 +31,31 @@ $ java -jar GameLiftLocal.jar -p 9080
 
 ```
 
-## 2. Start Server
+## 2. Set `sv_useGameLiftServer` to `true`
+
+To prevent the server from trying to reach out to Amazon GameLift while running locally, the AWS GameLift gem server manager will not initialize unless you opt-in to using it by setting the `sv_useGameLiftServer` CVAR to `true`. You can set this value by creating a settings registry file named **commands.MYPROJECT_serverlauncher.setreg** in the **YourProject/Registry** folder with the following contents:
+
+```json
+{
+    "Amazon":
+    {
+        "AzCore":
+        {
+            "Runtime":
+            {
+                "ConsoleCommands":
+                {
+                    "sv_useGameLiftServer": "true"
+                } 
+            } 
+        } 
+    } 
+}
+```
+
+This will let your game server use GameLift local when running in the Editor with `ctrl` + `g` or when running the server launcher directly (see next step).
```

Comment:
Nitpick: You might want to change this to "running in the Editor when entering game mode", I'm pretty sure control+g is a key binding that can be customized. If they're using the UI to enter/exit game mode by clicking the play arrow (in the top right, by default), this might not be as clear.

Also, how does this system work when entering the non-game simulation mode? That's the little bouncing ball that's to the top right (in the default UI layout) to the right of the enter/exit game mode arrow. It's a mode that initializes some game systems, but not all. Do servers get spun up, does multiplayer run in some fashion in simulation mode, or is most of that disabled? If it's all disabled, no need to mention it. If it is enabled, then it's probably worth including a note on that in the documentation here, too.